code/api/src/modules/user/mutations.js

```diff
@@ -3,7 +3,24 @@ import { GraphQLString, GraphQLInt } from 'graphql'
 
 // App Imports
 import { UserType } from './types'
-import { create, remove } from './resolvers'
+import { create, remove, update } from './resolvers'
```

Comment:
nice catch with this - it's been challenging to figure out what needs to be required/imported into various files since that was rarely (if ever) part of the process with rails